elements/pf-code-block/README.md

```diff
@@ -1,28 +1,29 @@
 # PatternFly Elements Code Block
-`<pfe-code-block>` is a component that contains two or more lines of ready-only 
+`<pf-code-block>` is a component that contains two or more lines of ready-only 
 code. The code in a code block can be copied to the clipboard.
 
 Read more about Code Block in the [PatternFly Elements Code Block 
 documentation](https://patternflyelements.org/components/code-block)
 
 ##  Installation
 
-Load `<pfe-code-block>` via CDN:
+Load `<pf-code-block>` via CDN:
 
 ```html
-<script src="https://jspm.dev/@patternfly/elements/pfe-code-block/pfe-code-block.js"></script>
+<scriptpf 
+src="https://jspm.dev/@patternfly/elements/-code-block/pf-code-block.js"></script>
```

Comment:
`elements/-code-block/pf-code-block.js` should be `elements/pf-code-block/pf-code-block.js`